README.md

```diff
@@ -155,7 +155,17 @@ new ProcessExecutor().command("java", "-version")
 
 <hr/>
 
-* Handling output line-by-line while process is running
+* Handling output line-by-line while process is running (java 8+)
+
+```java
+new ProcessExecutor().command("java", "-version")
+    .redirectOutput(LogOutputStream.create(line -> ...))
```

Comment:
Theoretically, there could be also a new method here `.processOutput(line -> ...)`, WDYT?